tests/test_orm.py

```
@@ -105,44 +110,51 @@ def test_orm_add_and_query(self):
-        self.assertEqual(len(queryset.all()), 2)
-        self.assertEqual(queryset.first().name, "user a")
-        self.assertEqual(queryset.first().age, 20)
+       # Generate a UID with validation
+        original_uids = set()
+        users = [
+            User(name="user a", age=20),
+            User(name="user b", age=30),
+            User(name="user c", age=40)
+        ]
+        for u in users:
+            original_uids.add(u.uid) 
```

Ruff (0.8.2)

Trailing whitespace

Remove trailing whitespace

(W291)